package.json

```diff
@@ -26,6 +26,7 @@
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
     "rrule": "2.6.4",
+    "source-map-explorer": "2.4.2",
```

Comment:
Could you clarify what you mean by separate `package.json`?

Reply:
Basically, you put the task into a separate dir, such as `build-systsem/tasks/coverage-map/...` and it will have its own `package.json` with all deps and so on. The `coverage-map` gulp task would need to run `yarn` in that dir before proceeding to executing the task.

Reply:
Please take a look at the changes I made here:
build-system/tasks/coverage-map/package.json

If it's okay, I'll go ahead and revert the changes in the root package.json.

Reply:
Yes. Looks great. Just couple more comments and, yes, let's revert the root `package.json`.

Reply:
What about the `package-lock.json`?

Reply:
I don't think there should be any `package-lock.json` at all - delete it, it might be there because you ran `npm install`. A `yarn.lock` should follow in lock step with each `package.json`. Thus, if a `package.json` has not changed, neither should `yarn.lock`. But, if you ever in doubt, you can simply delete `yarn.lock` and run `yarn` in the respective directory.

Reply:
Got it. This should be done now.